routes/api_v1_0.py

```diff
@@ -469,10 +470,14 @@ def get_data_v1_0():
 
         bearings = None
         if "mag" in result["variable"]:
+            bearings_var = DatasetConfig(
+                result["dataset"]).variable[result["variable"]].bearing_component
+            if not bearings_var:
+                bearings_var = "bearing"            
```

Comment:
You can use the `DatasetConfig` object that's already been instantiated on line 447! And you can also make use of how `or` short-circuits in Python to make the if-statement a one-liner:
```suggestion
            bearings_var = config.variable[result["variable"]].bearing_component or "bearing"   
```

Reply:
So if `config.variable[result["variable"]].bearing_component` is `None`, "bearing" will be assigned to `bearings_var`.